src/System.Memory/src/System/Buffers/Text/Base64Encoder.cs

```diff
@@ -228,6 +228,6 @@ private static int EncodeAndPadTwo(ref byte oneByte, ref byte encodingMap)
 
         private const byte EncodingPad = (byte)'='; // '=', for padding
 
-        private const int MaximumEncodeLength = (int.MaxValue >> 2) * 3; // 1610612733
+        private const int MaximumEncodeLength = (int.MaxValue / 4) * 3; // 1610612733
```

Comment:
Given this is a constant, the change is OK. However, in general, division by 4 is slower than bit shifting by 2.

![image](https://user-images.githubusercontent.com/6527137/38441455-a0280a34-3999-11e8-9f24-19dc4fa578b5.png)

```C#
    public int UseBitShift(int value) {
        return (value >> 2) * 3;
    }
    
    public int UseDivision(int value) {
        return (value / 4) * 3;
    }
```